### ISSUE TITLE
Return of Burned Tokens by Mistake

### ISSUE THREAD

**Issue by ICA3DaR5** [rchain/bounties#990]
[My Ethereum Address](https://etherscan.io/address/0x33678Cb73B27ad766FB4cF49C3b007E5C46d1a49) (which is also known to the bounty system and the co-op, since the beginning of my involvement.)

1. I got paid by the bounty system for the month of March the amount of [4,135.33 RHOC's](https://etherscan.io/tx/0xa87db2136df64e06dcf4afdbf0be49b4d2bc5c8181d7ede0d21c364b9d5a4885), then I mistakenly sent [3,135.33000018 RHOC's](https://etherscan.io/tx/0x308750e703503d1849f79a8988c55ba5ee7306a03e0577d88363188a0421f5e7) back to the [RchainToken Contract Address](https://etherscan.io/token/0x168296bb09e24a88805cb9c33356536b980d3fc5?a=0x168296bb09e24a88805cb9c33356536b980d3fc5)

2. I also mistakenly sent another [768.18181818 RHOC's](https://etherscan.io/tx/0x9efdf29f8daf210bd9955d676cfda045d92205d3a96d527bb0a50305c39d0052) back to the [Rchain Token (RHOC) Contract Address](https://etherscan.io/token/0x168296bb09e24a88805cb9c33356536b980d3fc5?a=0x168296bb09e24a88805cb9c33356536b980d3fc5)

Sending the RHOC's to the [RChainToken Contract Address](https://etherscan.io/token/0x168296bb09e24a88805cb9c33356536b980d3fc5?a=0x168296bb09e24a88805cb9c33356536b980d3fc5) essentially **burns** them.

**Total of RHOC's I've burned**: 3903.51181836 RHOC's

Since we are in a redemption process, I took the liberty to create this issue.

:+1: for endorsement, :-1: for objection supported by comments.

**Comment by tucsonblockchain** [comment]
Guys this is cryptocurrency, and while mistakes happen, remember this is cryptocurrency and Rchain should make every effort to pay him back. If Rchain can make a mistake and overpay and underpay people on bounties and expect members to pay it back because we are all one happy cooperative, it would be absolutely unacceptable to NOT pay ICA3DaR5 back on this one. How many millions of tokens were created out of thin air I may ask? Pay the man!

**Comment by ICA3DaR5** [comment]
@tucsonblockchain Thanks a lot for your support!

**Comment by jimscarver** [comment]
Arguments for why I, and others, feel this is the right thing to do are in the prior issue #753 

The accounts that reacted are not all listed  when your hover over the 👍 reaction to this issue.
To get the list of all the reactions paste in the following query at https://developer.github.com/v4/explorer/
```
{
  repository(owner: "rchain", name: "bounties") {
    issue(number: 990) {
      reactions(first: 100) {
        nodes {
          user {
            login
          }
          content
        }
      }
    }
  }
}

**Comment by Phistr90** [comment]
This issue isnt a proper bounty issue nor does it provide a clear list actions that are requested to be taken. Only the title gives  a hint. But technically there is no way to return Rhocs other than hard forking Ethereum. Besides that, it's impossible.
Furthermore, doing a mistakes once, okay  ... we all do mistakes. But doing the same mistake twice... 'Be your own bank' comes with responsibilities and at some point people need to start taking responsibility for their actions.
However, I guess that @ICA3DaR5 's intention was to propose a process that would indemnify the loss he experienced by sending the rhocs to the token contract address. There are  3 ways how such a process  might look like: 1) The coop pays for the loss  out of its treasury.  For this it would be reasonable to bring a proposal in for the annual membership meeting that defines a process by which entities that mistakenly sent funds to an address that is provably under nobody's control  are compensated by the coop.  2) The coop pays a loan to the entities that arguably can show that they lost control of the funds. The loan has only to be paid back if the funds are moved.  Also for this process it would be reasonable to bring in a proposal for the annual membership meeting 3) The compensation is done during the genesis process. This would require a social consensus beforehand.
Personally I am absolutely against option 3.  It would introduce a giant overhead in the overall process that is not remotely close to be justified  and would increase the overall attack surface a lot. I would be fine if @ICA3DaR5 case is covered by a proposed and by the membership accepted process  like 1 or 2.

**Comment by jimscarver** [comment]
See #753 for the original bounty issue. 

I suggest we need to be able to make decisions like this without a full member vote. If we cannot get consent from the engaged members and our leadership on a compromise on issues this small without engaging the entire membership we cannot begin to become an effective decentralized organization that scales.

@kennyrowe  is the one who suggested genesys would be the time to compensate David for the loss rather than a bounty issue. I think we should just pay him an amount voted in the bounty system where trusted member have the greatest voting power. My argument is that rhocs will be burned anyway and we have the power to make him whole. For me this is a test of whether RChain has a heart or not.  So far most reactions to this issue are :thumbsup: . 

If a member had a car accident and we had the medication he needed that was about to expire we would do the right thing I am sure. This is not so different. It costs us nothing to show that we value people over technicalities and are truely aiming at @leithaus  s vision of a human system that puts people first and rights wrongs when it has the power to do so.

As I stated in #753 there should be some consequence for making a mistake and 90% restitution might be reasonable. Each certified member can make our own judgement of how much to budget in the bounty system.

In so far as there may be future issues requesting restitution of lost funds each can be considered individually and will be supported by the membership or not. 

If we turn our back on members suffering loss makes me wonder if rchain is really creating a better world.

**Comment by tucsonblockchain** [comment]
I agree with @jimscarver in full because beyond that this project is nothing more than group think

**Comment by Phistr90** [comment]
What is this here? A cheering group? Those with the most friends will get compensated? I haven't seen any kind of arguments or proposals laid out. Could you please go ahead and formalize what you actually want to propose? Can you apply at least the SMART objectives?

**Comment by tucsonblockchain** [comment]
stick the proposal

**Comment by Cryptovideos** [comment]
exactly

**Comment by Cryptovideos** [comment]
and had this been you @Phistr90 id be waving my Pom Poms just as hard for you!

**Comment by jimscarver** [comment]
Perhaps  issue #990 should be labeled discussion until we decide a resolution  path, genesis or bounty. Perhaps issue #753 might be reopened and we just vote and hope we find a sponsor. That is an agile solution. Are there any objections?

I have a faith in humanity and think the underdog will be represented and those who express selfishness will lose trust and thus power in the collective. I have not found any members yet that do not have a good heart and can be offboarded from the competitive world into the cooperative world recognizing that what benefits the whole benefits us all and that our success depends on the success of the collective. 

John Kelden and others profess that all decisions are made either out of love or fear. Those made out of love are enriching where as those make out of fear are demoralizing. My view is that for rchain to be great it must apply practical wisdom, not heartless rules.  https://www.ted.com/talks/barry_schwartz_using_our_practical_wisdom

I suggest that objection to restitution perpetuates the ethics of a zero sum game which may forstall our blossoming.

**Comment by jimscarver** [comment]
We can vote an amount in #753 and decide later whether pay through the bounty system or in the genesis block. @kennyrowe suggested the genesis block but that would clearly involve more work. I see no justification for that.

**Comment by jimscarver** [comment]
The september payment for bounties were sent to the wrong people accidently. Member who recieved too much have been asked to return the difference.

I feel there would be some sort of hypocrisy in the members having to make good of the coops mistake but turning a blind eye to the coop making good on a members mistake.

**Comment by ICA3DaR5** [comment]
@Phistr90 I am just trying to retrieve my RHOC back, due to a human error.

**Comment by jasoncruzzy** [comment]
@ICA3DaR5  you should be paid back, if actually his claims can be verified by the coop. This issue has lingered for too long

**Comment by ICA3DaR5** [comment]
@jasoncruzzy 

> @ICA3DaR5 you should be paid back, if actually his claims can be verified by the coop. This issue has lingered for too long

My claims can be verified by the co-op and by anyone for that matter. I sent the RHOC to the RChain Contract Address NOT on another regular address.

**Comment by philipandri002** [comment]
@phistr90 this is not about how many friends you have or don't have, this is about fairness and commitment from the coop to its members, I assume we are all friends here. @ICA3DaR5 has been very committed to this community, also don't forget this can happen to anybody.

**Comment by entropee** [comment]
These are acceptible losses for a project this size. Could we attribute it to breakage or something. Didn't this particular mistake happen as a consequence of another mistake on the coop's part? It's the nature of the territory. Losses of this type at this stage are bound to happen. I know I sweated bullets until I had repaid what had mistakenly been sent to me by the coop which was a similar amount to what's being discussed here. I would have expected the coop to swallow the loss if I had screwed things up although I'm glad in my case in went well. But I would expect the coop to return to me what is mine, as well, if that was possible.  I would expect the coop to make good on my losses if I I could show that I had taken reasonable precautions in attempting to make to coop whole and my losses came about in the process of of that effort. IANAL.

**Comment by ICA3DaR5** [comment]
Any updates on this?

**Comment by ICA3DaR5** [comment]
I can see [here](https://www.rchain.coop/blog/rchain-2018-election-results/) that 410,663,574 RHOC will be burned. Can we count my burned RHOC in that sum?
